src/App/Documentation/components/MediaObject/index.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+
+import { ComponentPreview, DocToc } from "#";
+import MediaObjectComponent from "@/MediaObject";
+
+const Overview = () => (
+    <>
+        <h2 id="overview">Overview</h2>
+        <p>Use the <code className="token property">.media</code> class on an object to style it as a small container that feature a left- or right-aligned image/icon alongside textual content.</p>
+        <ComponentPreview language="html" showCasePanel={true} codeFigure={true}>
+            <MediaObjectComponent imgUrl="https://via.placeholder.com/48x48" heading="Bob Corlsan" text="bob.corlsan@payex.com" />
+            <MediaObjectComponent icon="account_circle" heading="Bob Corlsan" text="bob.corlsan@payex.com" />
+        </ComponentPreview>
+    </>
+);
+
+const Sizes = () => (
+    <>
+        <h2 id="sizes">Sizes</h2>
+        <p>Sizes for media object...</p>
+        <h3>Small</h3>
+        <p>Small media object <code className="token property">.media-sm</code>...</p>
+        <ComponentPreview language="html" showCasePanel={true} codeFigure={true}>
+            <MediaObjectComponent size="sm" imgUrl="https://via.placeholder.com/40x40" heading="Bob Corlsan" text="bob.corlsan@payex.com" />
+            <MediaObjectComponent size="sm" icon="account_circle" heading="Bob Corlsan" text="bob.corlsan@payex.com" />
+        </ComponentPreview>
+        <h3>Large</h3>
+        <p>Large media object <code className="token property">.media-lg</code>...</p>
+        <ComponentPreview language="html" showCasePanel={true} codeFigure={true}>
+            <MediaObjectComponent size="lg" imgUrl="https://via.placeholder.com/56x56" heading="Bob Corlsan" text="bob.corlsan@payex.com" />
+            <MediaObjectComponent size="lg" icon="account_circle" heading="Bob Corlsan" text="bob.corlsan@payex.com" />
+        </ComponentPreview>
+    </>
+);
+
+const MediaAlignment = () => (
+    <>
+        <h2 id="media-alignment">Media alignment</h2>
```

Comment:
Same comment as previous review: A lot of components already use "position" within their classes when there's a choice of alignment or positioning. Should this one aswell?